programmers2/게임-맵-최단거리.js

```diff
@@ -0,0 +1,29 @@
+function solution(maps) {
+    const yLen = maps.length;
+    const xLen = maps[0].length;
+    const dx = [-1, 1, 0, 0];
+    const dy = [0, 0, -1, 1];
+    const queue = [[0, 0]];
+    
+    while (queue.length) {
+        const xy = queue.shift();
+        const x = xy[0];
+        const y = xy[1];
```

Comment:
```suggestion
        const [x, y] = queue.shift();
```
큐의 요소가 배열 형태여서 구조 분해 할당 방식으로 선언해도 됩니당

그리고 x와 y의 순서를 일관되게 하면 더 좋을 것 같아요.